src/deckardcain.js

```diff
@@ -20,7 +20,7 @@ const REFRACT_PARSE_RESULT_ELEMENT_YAML = /[\uFEFF]?\n?\s*element: ?["']parseRes
  * @returns {string|null} Media type of given file.
  */
 function identify(source) {
-  if (source.match(API_BLUEPRINT_HEADER)) {
+  if (source.substring(0, 11).match(API_BLUEPRINT_HEADER)) {
```

Comment:
+1 to being super suspicious
On 1 Dec 2015 17:10, "Ondrej Musil" <notifications@github.com> wrote:

> In src/deckardcain.js
> <https://github.com/apiaryio/deckardcain/pull/21#discussion_r46297872>:
>
> > @@ -20,7 +20,7 @@ const REFRACT_PARSE_RESULT_ELEMENT_YAML = /[\uFEFF]?\n?\s*element: ?["']parseRes
> >   * @returns {string|null} Media type of given file.
> >   */
> >  function identify(source) {
> > -  if (source.match(API_BLUEPRINT_HEADER)) {
> > +  if (source.substring(0, 11).match(API_BLUEPRINT_HEADER)) {
>
> Ok then I trust you. *(but not to users)*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiaryio/deckardcain/pull/21/files#r46297872>.
>